### DISCUSSION TITLE
Is it a bug that I can't use the new touch input with a mouse?

### DISCUSSION THREAD

**Discussion by Schweeeeeeeeeeeeeeee** [ppy/osu#22834] [Q&A]
I'm trying to play osu on my phone with a mouse and use the touch screen as buttons but when I hold the mouse button down after clicking a circle and try to use the touch for clicking after that it won't work.

**Comment by ItsShamed** [comment]
I don't think you're supposed to mix input types. If you are playing with the mouse (probably using OTG) you can look into using the mouse clicks while you're at it. IIRC, https://github.com/itdelatrisu/opsu doesn't work well either at mixing input types.
The only mix that would make sense is touch + keyboard, but I haven't tested if it works well.

**Comment by peppy** [comment]
I'm not sure we can support this kind of multi-device play when it comes to touch+something.
